app/views/feedback/steps/_purpose.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1 class="govuk-label-wrapper">
+  <label class="govuk-label">Share your feedback</label>
+</h1>
+
+<p>
+  We'll review this feedback to make our website better. Answer 3 questions to let us know how we can improve.
+</p>
+
+<p>All feedback is anonymous and we cannot respond to individual queries. </p>
+
+<p>If you have a question or need help, please visit our get <%= link_to "help and support page.", get_help_and_support_path %></p>
```

Comment:
```suggestion
<p>If you have a question or need help, please visit our get <%= link_to "help and support page"., get_help_and_support_path %></p>
```
(Our style is full-stops outside link text - sorry, didn't spot before!)

Reply:
Also - this link doesn't work!

It seems to be looking for `get-help-and-support` but the slug of that page is `help-and-support`

Is this path used anywhere else?